packages/gatsby-source-wordpress/README.md

```diff
@@ -140,10 +140,10 @@ module.exports = {
         // URLs for thumbnails, or any other media detail.
         // Defaults to false
         keepMediaSizes: false,
-        // use a custom normalizer which is applied after the built-in ones.
-        normalizer: function({ entities }) {
-          return entities
-        },
```

Comment:
I don't think we should remove this from documentation completely. People might be using that and it will still conitnue to work. This is essentially simplified version of `normalizers` (with less power), so maybe we could just add mention that if you need more control over order of normalizers (internals / custom one) - to use `normalizers`